examples/deploy-contract/index.js

```diff
@@ -5,35 +5,34 @@ const { PlatformClient } = require('platform-deploy-client');
 const artifactFile = require('./artifacts/Box.json');
 
 async function main() {
-    const creds = { apiKey: process.env.ADMIN_API_KEY, apiSecret: process.env.ADMIN_API_SECRET };
-    const client = new PlatformClient(creds);
-
-    await client.BlockExplorerApiKey.create({
-        network: "goerli",
-        key: "AAABBBCCC..." // Add your Block Explorer API Key here
-    });
-
-    const keys = await client.BlockExplorerApiKey.list();
-    console.log(keys);
-
-    // Get approval process for deployment on Goerli
-    const config = await client.Deployment.getApprovalProcess("goerli");
-    console.log(config);
-
-    const deployment = await client.Deployment.deploy({
-        contractName: 'Box',
-        contractPath: 'contracts/Box.sol',
-        network: 'goerli',
-        artifactPayload: JSON.stringify(artifactFile),
-        licenseType: 'MIT',
-        salt: "2",
-        verifySourceCode: true,
-    });
-
-    const deploymentStatus = await client.Deployment.get(deployment.deploymentId);
-    console.log(deploymentStatus);
+  const creds = { apiKey: process.env.ADMIN_API_KEY, apiSecret: process.env.ADMIN_API_SECRET };
+  const client = PlatformClient(creds);
+
+  await client.BlockExplorerApiKey.create({
+    network: 'goerli',
+    key: 'AAABBBCCC...', // Add your Block Explorer API Key here
+  });
+
+  const keys = await client.BlockExplorerApiKey.list();
+  console.log(keys);
+
+  // Get approval process for deployment on Goerli
```

Comment:
Should we switch to Sepolia while we are here?

Reply:
Sure. Should probably merge this one first then: https://github.com/OpenZeppelin/defender-client/pull/230